capstone/src/main/resources/static/map.js

```diff
@@ -11,6 +11,15 @@ async function createMap() {
    jsonEvents.forEach(event => createMarker(event, campusMap));
   }
 
+function createMapForASingleEvent(event) {
+  var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
```

Comment:
you can use the code I pushed to show the user's selected university instead

Reply:
Yeah I just realized this branch is behind